src/widget.ts

```diff
@@ -70,54 +39,15 @@ abstract class BabylonBaseView extends DOMWidgetView {
   inspector = this.values.inspector;
 
   protected resizeCanvas(): void {
```

Comment:
Is this method redundant? It also exists in core package and is called upon render.

Reply:
Tbh i don't see anywhere calling it, maybe @MargrietGroenendijk can answer about this.

Reply:
It has been moved to the core package